modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -105,48 +142,75 @@
   "image": "Image",
   "imageFile": "Image File",
   "imageTag": "Image Tag",
+  "imageTags": "Image Tags",
   "images": "Images",
+  "insertAndRedirect": "{{ saveLabel }} {{ typeLabel }} and be redirected to the {{ typeLabel }}.",
+  "insertAndReturn": "{{ saveLabel }} and return to the {{ typeLabel }} listing.",
+  "insertAndNew": "{{ saveLabel }} {{ typeLabel }} and create a new one.",
   "italic": "Italic",
   "lastEdited": "Last Edited",
   "leavePageDescription": "The content you're trying to edit belongs to another document and must be edited there.\nChanges made to {{ oldTitle }} are saved automatically.",
   "leavePageHeading": "Leave {{ oldTitle }} to edit {{ newTitle }}?",
   "link": "Link",
   "listJoiner": ", ",
   "live": "Live",
+  "lastUpdatedBy": "Last saved on {{ updatedAt }} <br /> by {{ updatedBy }}",
```

Comment:
Should this have markup? Might cause issues in responsive layouts.

Reply:
Feel free to open a ticket on that, but it's not new to this PR (it's just been migrated here so it can be localized).